packages/expo-doctor/src/doctor.ts

```diff
@@ -172,54 +172,66 @@ export function getChecksInScopeForProject(exp: ExpoConfig, pkg: PackageJSONConf
 export async function actionAsync(projectRoot: string) {
   await warnUponCmdExe();
 
-  const projectConfig = getConfig(projectRoot);
-
-  // expo-doctor relies on versioned CLI, which is only available for 44+
   try {
-    if (ltSdkVersion(projectConfig.exp, '46.0.0')) {
-      Log.exit(
-        chalk.red(`expo-doctor supports Expo SDK 46+. Use 'expo-cli doctor' for SDK 45 and lower.`)
-      );
+    const projectConfig = getConfig(projectRoot);
+
+    // expo-doctor relies on versioned CLI, which is only available for 44+
+    try {
```

Comment:
I don't think this extra try/catch is needed? Any error in determining the version would be caught by the outer try/catch I think.